pkg/sql/schemachanger/scbuild/internal/scbuildstmt/create_index.go

```diff
@@ -212,6 +215,131 @@ func nextRelationIndexID(b BuildCtx, relation scpb.Element) catid.IndexID {
 	}
 }
 
+func newUndefinedOpclassError(opclass tree.Name) error {
```

Comment:
Question: what is an operator class?

Reply:
It's mostly for inverted indexes: https://www.postgresql.org/docs/current/gin-builtin-opclasses.html
But looks like only geometry and trigram are taken care of in crdb: https://github.com/cockroachdb/cockroach/blob/b6e661e4ff03bd7c9650c703f1fb98dc213cad63/pkg/sql/create_index.go#L367-L367

Reply:
@Xiang-Gu https://www.postgresql.org/docs/current/indexes-opclass.html, it pretty much determines how values are compared / stored for a given index. I'll make add comments in the areas where this is checked:

// OpClass in Cockroach is used to determine how values will be compared/stored within the index,
// similar to Postgres.